pkgs/applications/blockchains/monero/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
     boost miniupnpc openssl unbound
     cppzmq zeromq pcsclite readline
     libsodium hidapi randomx rapidjson
-    pythonProtobuf libusb
```

Comment:
Are you sure about this? The code seems to be looking for `libusb-1.0`:

https://github.com/monero-project/monero/blob/28e0a28b96212abce237e7b94df78bd4541a66c1/cmake/FindLibUSB.cmake#L21

(same for monero-gui)

Reply:
Then it must be broken on master.